cs/configuring.texy

```diff
@@ -86,11 +87,20 @@ $configurator->addConfig(__DIR__ . '/config/config.neon');
 
 Pokud chceme přidat více nezávislých souborů, můžeme funkci `addConfig()` zavolat vícekrát.
 
+/--php
+$configurator->addConfig(__DIR__ . '/config/config.neon');
+$configurator->addConfig(__DIR__ . '/config/config.local.neon');
+\--
 
-Přidání parametrů
------------------
 
-Parametry do naší aplikace můžeme dostat i přes metodu `addParameters()`.
+Normální parametry
+------------------
+
+Parametry do naší aplikace můžeme nastavit metodou `addParameters()`, hodnoty těchto parametrů musí být pro
```

Comment:
to jsou parametry do DI kontejneru, případně configur, nikoliv do aplikace. Rovnou tam zmiň, že na parameter `foo` se pak v konfiguraci odkáže zápisem `%foo%`.